cri/v1alpha2/cri_utils.go

```diff
@@ -256,8 +255,9 @@ func makeSandboxPouchConfig(config *runtime.PodSandboxConfig, image string) (*ap
 	hc := &apitypes.HostConfig{}
 
 	// Apply runtime options.
-	if annotations := config.GetAnnotations(); annotations != nil {
-		hc.Runtime = annotations[anno.KubernetesRuntime]
+	// NOTE: whether to add UntrustedWorkload
+	if runtimehandler != "" {
```

Comment:
Just assign `runtimehandler` to `hc.Runtime`, no need to judge it first.

The underlying code will handle it if it is empty.

Reply:
Maybe double check is not bad. ^_^ WDYT?

Reply:
What's more, `hc.Runtime` has not been initialized explicitly, its value is `""` by default. The judge here is totally redundant.